contrib/benchmarking_nd4j/pom.xml

```diff
@@ -85,6 +88,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
ideally you'd use the following here instead of half hard-coding values.
```
<source>${java.version}</source>
<target>${javac.target}</target>
<compilerVersion>${java.compiler.version}</compilerVersion>
```